### ISSUE TITLE
Run supervised LSTM-based model with default hyperparameters yield an F1 of around 0.46

### ISSUE THREAD

**Issue by YuFeng0023** [williamleif/GraphSAGE#46]
Hi:
I used this command : 
`python -m graphsage.supervised_train --train_prefix ./example_data/ppi --model graphsage_seq --sigmoid`
and I get this result:
`Optimization Finished!
Full validation stats: loss= 0.52820 f1_micro= 0.46528 f1_macro= 0.23076 time= 0.33810
Writing test set stats to file (don't peak!)`
Is default hyperparameters the optimal parameters？The result(f1_micro= 0.46) is far from the result(f1_micro=0.61) listed in the paper.
I set epoch=50, and I get the results range from 0.60 to 0.64. So, what are the optimal parameters for supervised LSTM model on the PPI?

**Comment by RexYing** [comment]
Hi,
Have you downloaded the complete data from our website? See http://snap.stanford.edu/graphsage/ for the PPI data.

With adam, the learning rate should be around 0.0005

Also, you can generate the walks file with the deepwalk implementation.

It should converge in ~10 epochs.

**Comment by EtoDemerzel0427** [comment]
> Hi,
> Have you downloaded the complete data from our website? See http://snap.stanford.edu/graphsage/ for the PPI data.
> 
> With adam, the learning rate should be around 0.0005
> 
> Also, you can generate the walks file with the deepwalk implementation.
> 
> It should converge in ~10 epochs.

Hi, 

I don't really get your point here, since he said he was trying on supervised version, and walks file should only be used in unsupervised version I think.

Also, after tuning the learning rate here to 0.0005(is this really for supervised version again?), my result is not so good.
